packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -274,7 +274,12 @@ const _BaseLink: React.ForwardRefRenderFunction<BladeElementRef, BaseLinkProps>
 ): ReactElement => {
   const [isVisited, setIsVisited] = useState(false);
   const childrenString = getStringFromReactText(children);
-  const { currentInteraction, setCurrentInteraction, ...syntheticEvents } = useInteraction();
+  const {
+    currentInteraction,
+    setCurrentInteraction,
+    onBlur: onBlurInteraction,
```

Comment:
why do we need onBlur interaction?

Reply:
The `useInteraction` hook needs this for resetting the state to `default` when the link is blurred. This was earlier being passed but in some recent refactor got missed.

Have also added a regression test to catch these in the future in this PR.